pysal/contrib/viz/plot.py

```diff
@@ -9,38 +9,44 @@
 import pysal as ps
 import matplotlib.pyplot as plt
 
-
-def moran(var, w, xlabel='', ylabel='', title='', custom=(5,5)):
-	'''
+def mplot(m, xlabel='', ylabel='', title='', custom=(7,7)):
+    '''
     Produce basic Moran Plot 
     ...
-    Arguments
+    Parameters
     ---------
-    var	            : array
-                      values of variable
-    w       	    : spatial weight
+    m            : array
```

Comment:
Not sure I understand your view. Mine was really that we need to give the user some guidance as to what to pass in as `m` for the function. Right now, it's asking for a (numpy) `array`, but what it really takes is the output of `ps.Moran`, hence my suggestion to use that in the documentation.